File: SimulationPackage.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;


public class SimulationPackage
{
    public string jsonString;
    public SimulationPackageJSON json;
    public SimulationPackageState state = SimulationPackageState.None;

    public SimulationPackage(string jsonString)
    {
        this.jsonString = jsonString;
        this.json = JsonUtility.FromJson<SimulationPackageJSON>(jsonString);
        if (this.json == null)
            state = SimulationPackageState.Invalid;
        else
            state = SimulationPackageState.Valid;
    }
}

public enum SimulationPackageState
{
    None,Valid,Invalid
}

[System.Serializable]
public class SimulationPackageJSON
{
    public string topic;
    public string payloadJson;
    public int signature;
}

File: SimulationPackage.py
import json

class SimulationPackage:
    def __init__(self,jsonString = "{}"):
        self.jsonString = jsonString
        self.simulationPackageJSON = json.loads(jsonString)
        if(self.simulationPackageJSON == None):
            self.state = "Invalid"
        else:
            self.state = "Valid"
            self.topic = self.simulationPackageJSON["topic"]
            self.payloadJson = self.simulationPackageJSON["payloadJson"]
            self.signature = self.simulationPackageJSON["signature"]